templates/bundle/resources/usr/local/bin/monitor_liferay_lifecycle.sh

```diff
@@ -110,9 +117,9 @@ function main {
 				) | telnet 127.0.0.1 11311 2> /dev/null
 			)
 
-			local active_count=$(echo "${telnet_content}" | grep -E "${LIFERAY_CONTAINER_STATUS_ACTIVE_MODULES}" | grep -c ACTIVE)
+			local active_count=$(echo "${telnet_content}" | grep -v "${LIFERAY_CONTAINER_STATUS_IGNORE_ACTIVE_MODULES}" | grep -E "${LIFERAY_CONTAINER_STATUS_ACTIVE_MODULES}" | grep -c ACTIVE)
```

Comment:
@chinmay-abhyankar, considering that a fragment never transitions to the status `Active`, please see https://liferay.slack.com/archives/C07GFPNS1AT/p1725532590516359, and it remains in the status `Resolved`, does it make sense to set by default the value `fragment` in the variable `LIFERAY_CONTAINER_STATUS_IGNORE_ACTIVE_MODULES`?

Reply:
Generally developers uses fragment in symbolic name of OSGI modules which are use to fragment OOTB modules. So to sent default is to ignore checking of all modules which have word fragment, even if the envionment variable is not set

Reply:
I see, @chinmay-abhyankar. Thanks for your clarification! :slightly_smiling_face: 

Since it seems to be a decision on the developer's side, maybe we should use the content of the variable `LIFERAY_CONTAINER_STATUS_IGNORE_ACTIVE_MODULES` only if it's set, does it make sense to you?